browser.jsx

```diff
@@ -1,9 +1,9 @@
 'use strict'
-var remote = require('remote')
-var Menu = remote.require('menu')
-var MenuItem = remote.require('menu-item')
-var clipboard = require('clipboard')
-var urllib = require('url')
+var remote = require('electron').remote
+var Menu = require('electron')
```

Comment:
Hi @myfrndjk, yes I get that same error and I specifically mentioned it in the initial description of the PR :)

I didn't have the time to debug it any further. Single tab browsing works ok :)